L1Trigger/L1THGCalUtilities/test/testHGCalL1T_multialgo_V16_cfg.py

```diff
@@ -74,12 +75,11 @@
 ## VFE
 chains.register_vfe("Floatingpoint", vfe.CreateVfe())
 ## ECON
-chains.register_concentrator("Supertriggercell", concentrator.CreateSuperTriggerCell())
 chains.register_concentrator("Threshold", concentrator.CreateThreshold())
-chains.register_concentrator("Bestchoice", concentrator.CreateBestChoice())
-chains.register_concentrator("AutoEncoder", concentrator.CreateAutoencoder())
+chains.register_concentrator("Bcstc", concentrator.CreateMixedFeOptions())
```

Comment:
Could you point out where the location of the configuration for the mixed bc+stc?  I couldn't quite follow the path to it from here.

Also another comment that's not particularly related to this PR in particular - should we update the default ECON-T algorithm to use bc+stc now/soon, so that we are at least using the correct (or more correct) TCs in the stage 2 emulation?  Or is there something preventing us from doing this?

Reply:
BC+STC `CreateMixedFeOptions()` config is imported from:
```py
import L1Trigger.L1THGCalUtilities.concentrator as concentrator
``` 
Then parameters are defined in `L1Trigger/L1THGCal/python/l1tHGCalConcentratorProducer_cfi.py` in the `custom_conc_proc` PSet.

Concerning the update to default BC+STC, the link mapping used is still a threshold-based "PU-driven" mapping, and not the link mapping planned to be used with BC ("signal-driven"), so if we switch to BC+STC now, it would be based on an "incorrect" link mapping.